src/dal/static/autocomplete_light/autocomplete.init.js

```diff
@@ -60,7 +60,7 @@ element was cloned with data - which should be the case.
         if (document.cookie && document.cookie != '') {
             var cookies = document.cookie.split(';');
             for (var i = 0; i < cookies.length; i++) {
-                var cookie = jQuery.trim(cookies[i]);
+                var cookie = yl.jQuery.trim(cookies[i]);
```

Comment:
hi,@jpic. upgrade.

but strictly speaking for now,this PR's real target is improve this file code style. the old topic is not right.

because finally, I found this error is cause by myself.I am so sorry.
I used django-autocomple-light  in others project with django 1.8,they works well . 
I create a new project with django1.9. but I load `dal` and `dal_select2` after `django.contrib.admin`
so error caused. and I found this file use jQuery not $ or yl.jQuery.

then I compare this project with others project in browser. I found jquery.init.js is different.
 
the truth is I compare jquery.init.js file from dal package  with  django.contrib.admin package's  jquery.init.js file. so, I trapped 😫...